daemon/notify.go

```diff
@@ -72,9 +70,9 @@ func notify(id int64, body string) error {
 	cmd.Env = append(cmd.Env, "DISPLAY=:0.0")
 	cmd.Env = append(cmd.Env, dbusAddr)
 	cmd.SysProcAttr = &syscall.SysProcAttr{Credential: &syscall.Credential{Uid: uint32(id)}}
-	out, err := cmd.CombinedOutput()
+	err := cmd.Run()
```

Comment:
Because this is a bit tricky `CombinedOutput` vs `Run` with timeout, would it make sense to have a function that runs the command to have the explanation? if not maybe add comment here why is `Run` used?